lib/foreman_chef/engine.rb

```diff
@@ -33,6 +33,10 @@ class Engine < ::Rails::Engine
       ActionView::Base.send :include, ChefProxyForm
     end
 
+    initializer 'foreman_chef.dynflow_world' do |app|
+       ForemanTasks.dynflow.require!
```

Comment:
foreman_tasks is not (yet) available for Debian/Ubuntu, I guess we're out of luck here for the moment?